src/structures/kmq_member.ts

```diff
@@ -0,0 +1,27 @@
+import Eris from "eris";
+import { getUserTag } from "../helpers/discord_utils";
+
+export default class KmqMember {
```

Comment:
Would it make sense to instead reuse player.ts here? A lot of the properties are shared, you could just move the new method over there

Reply:
We can look into making Player inherit this later